staging/publishing/rules.yaml

```diff
@@ -1085,6 +1053,8 @@ rules:
         branch: release-1.14
       - repository: apimachinery
         branch: release-1.14
+      - repository: apiserver
+        branch: release-1.14
```

Comment:
Argh, this is why the bot fails now.

Reply:
yep

Reply:
Why does the cloud-provider repo depend on apiserver?

Reply:
Reason are feature gates. With https://github.com/kubernetes/kubernetes/pull/74021 everything will improve.

Reply:
https://github.com/kubernetes/kubernetes/blob/7bcdbc5a51bcb5e856c0dd0d10b6a81d70da1358/staging/src/k8s.io/cloud-provider/features/gce.go#L19-L26